### DISCUSSION TITLE
Forge alternnative?

### DISCUSSION THREAD

**Discussion by Melyns** [lllyasviel/stable-diffusion-webui-forge#2455] [General]

**Comment by abzaloff** [comment]
> forge seems to be broken now so im looking for a web ui that can run flux (other than comfy). Thank you

Forge works great with Flux

**Comment by AIvisualz** [comment]
Forge is not broken and it runs better with Flux than Comfy (for me on 4090).
It lacks controlnet support for Flux, but to be honest it's not that good. For now you can create an image with controlnet using SDXL and then run it on flux with img2img.
It would be nice with redux support.
